app/src/main/res/layout/messages_layout.xml

```diff
@@ -69,37 +69,36 @@
         app:layout_scrollFlags="scroll|enterAlways">
 
         <RelativeLayout
-          android:layout_marginStart="@dimen/grid_new_3"
-          android:layout_marginEnd="@dimen/grid_new_3"
-          android:layout_marginLeft="@dimen/grid_new_3"
-          android:layout_marginRight="@dimen/grid_new_3"
+          android:paddingStart="@dimen/grid_new_3"
+          android:paddingEnd="0dp"
```

Comment:
took the padding off the parent so the `View pledge` button's background ripple goes all the way to the edge